python/tests/scripts/d2_higherfunc.metta

```diff
@@ -168,12 +168,14 @@
 !(assertEqualToResult
   (get-type (fmap (curry-a + 2) (Left "5")))
   ())
-!(assertEqualToResult
-  (get-type (fmap (curry-a + 2) (UntypedC "5")))
-  ())
-!(assertEqualToResult
-  (get-type (fmap (curry-a + 2) (UntypedC (Null) 5)))
-  ())
+; TODO: Two examples below are type-checked successfully because, (UntypedC "5")
+; can return result which has an appropriate type.
```

Comment:
```suggestion
; TODO: Two examples below are type-checked successfully because, (UntypedC "5")
; can return result which has an appropriate type. Uncomment when old_interpreter 
; feature is removed.
```